src/data/locales/en/translation.json

```diff
@@ -277,6 +277,7 @@
         "workOrEarn": "Did you work or earn money, <strong>whether you were paid or not</strong>? If self-employed, report earnings during the week you receive the money.",
         "help-workOrEarn": "Look at the date each week begins and ends. If you performed any work during this week, the earnings must be reported regardless if you received payment or not. This does not include self-employment.",
         "recentDisaster": "Are you unemployed as a direct result of a recent disaster (for example: COVID-19, earthquake, flood, mudslide, or fire) in California?",
+        "disasterChoice": "Select the option for how you were impacted by the COVID-19 pandemic.",
```

Comment:
Having a redundant key here (disasterChoice is the same as recentDisasterChoice below) seems like a lesser evil than disrupting the user experience when deploying. We can remove this on a following deploy.

Reply:
The current build in staging (with partial save and shorter confirmation number) shouldn't have the CDN cache issue (we serve the translations directly from node). It should be safe to just change the key. But if you want to be cautious, this is also fine.

Reply:
Oh I think I misunderstood @sharonwarner then... I thought she said that change was rolled back due to some issue with it, and the translation.json caching issue was still a problem. OK!